policy/engine.go

```diff
@@ -519,15 +529,24 @@ func (e *Engine) query(ctx context.Context, input interface{}, query string) (ou
 }
 
 func isWarning(rule string) bool {
-	warningRegex := regexp.MustCompile("^warn(_[a-zA-Z0-9]+)*$")
 	return warningRegex.MatchString(rule)
 }
 
 func isFailure(rule string) bool {
-	failureRegex := regexp.MustCompile("^(deny|violation)(_[a-zA-Z0-9]+)*$")
 	return failureRegex.MatchString(rule)
 }
 
+func problematicIf(moduleBody string) string {
```

Comment:
I liked the way we bandaid here, additionally, it might be useful to include https://github.com/open-policy-agent/opa/issues/6509 as a comment for posterity

Reply:
Added.